src/test/java/net/technolords/micro/command/StatsCommandTest.java

```diff
@@ -18,11 +23,13 @@
 import net.technolords.micro.config.ConfigurationManager;
 import net.technolords.micro.registry.MockRegistry;
 
-public class StatsCommandTest {
+public class StatsCommandTest extends RouteTestSupport {
     private static final Logger LOGGER = LoggerFactory.getLogger(StatsCommandTest.class);
     private Main main;
     private ProducerTemplate producerTemplate;
     private String availablePort;
```

Comment:
Move the part and logic of available port into the parent class (i.e. TestSupport)